python/pyspark/ml/tests.py

```diff
@@ -112,14 +112,15 @@ def test_pipeline(self):
         self.assertEqual(6, dataset.index)
 
 
-class TestParams(HasMaxIter, HasInputCol):
+class TestParams(HasMaxIter, HasInputCol, HasSeed):
     """
-    A subclass of Params mixed with HasMaxIter and HasInputCol.
+    A subclass of Params mixed with HasMaxIter, HasInputCol and HasSeed.
     """
 
-    def __init__(self):
+    def __init__(self, seed=None):
         super(TestParams, self).__init__()
         self._setDefault(maxIter=10)
+        self._set(seed=seed)
```

Comment:
It should probably do what other subclasses of Params do.  E.g., look at Word2Vec in features.py:
```
        kwargs = self.__init__._input_kwargs
        self.setParams(**kwargs)
```
This way, Param "seed" will only be explicitly set in self._paramMap if it is specified by the user.  We want to be able to distinguish between users specifying a parameter vs. not.